plugins/Goals/lang/en.json

```
@@ -3,63 +3,63 @@
         "AddGoal": "Add Goal",
         "AddNewGoal": "Add a new Goal",
-        "AddNewGoalOrEditExistingGoal": "%1$sAdd a new Goal%2$s or %3$sEdit%4$s existing Goals",
-        "AllowGoalConvertedMoreThanOncePerVisit": "Allow Goal to be converted more than once per visit",
+        "AddNewGoalOrEditExistingGoal": "%1$sAdd a new goal%2$s or %3$sEdit%4$s existing goals",
+        "AllowGoalConvertedMoreThanOncePerVisit": "Allow goal to be converted more than once per visit",
         "AllowMultipleConversionsPerVisit": "Allow multiple conversions per visit",
-        "BestCountries": "Your best converting countries are:",
-        "BestKeywords": "Your top converting keywords are:",
-        "BestReferrers": "Your best converting websites referrers are:",
-        "BestEntryPage": "Your best converting entry page is:",
-        "CaseSensitive": "Case sensitive match",
+        "BestCountries": "The countries with the most conversions are:",
+        "BestKeywords": "The keywords with the most conversions are:",
+        "BestReferrers": "The websites sending the most referrers are:",
+        "BestEntryPage": "The entry page with the most conversions is:",
+        "CaseSensitive": "Case-sensitive match",
         "CancelAndReturnToGoals": "Cancel and %1$sreturn to the list of goals%2$s",
         "CategoryTextGeneral_Visitors": "User location",
         "CategoryTextReferrers_Referrers": "Referrers",
         "CategoryTextVisitsSummary_VisitsSummary": "User attribute",
         "CategoryTextDevicesDetection_DevicesDetection": "Devices",
         "CategoryTextGeneral_Actions": "Pages",
         "CategoryTextGeneral_Visit": "engagement",
-        "ClickOutlink": "Click on a Link to an external website",
+        "ClickOutlink": "Click on a link to an external website",
         "SendEvent": "Send an event",
-        "ColumnAverageOrderRevenueDocumentation": "Average Order Value (AOV) is the total revenue from all Ecommerce Orders divided by the number of orders.",
+        "ColumnAverageOrderRevenueDocumentation": "Average order value (AOV) is the total revenue from all e-commerce orders divided by the number of orders.",
         "ColumnAveragePriceDocumentation": "The average revenue for this %s.",
-        "ColumnAverageQuantityDocumentation": "The average quantity of this %s sold in Ecommerce orders.",
-        "ColumnConversionRateDocumentation": "The percentage of visits that triggered the goal %s.",
-        "ColumnConversionRatePageViewedBeforeDocumentation": "The percentage of all conversions for goal %s where this page was viewed before conversion.",
+        "ColumnAverageQuantityDocumentation": "The average quantity of this %s sold in e-commerce orders.",
+        "ColumnConversionRateDocumentation": "The percentage of visits that triggered the %s goal.",
+        "ColumnConversionRatePageViewedBeforeDocumentation": "The percentage of all conversions for the %s goal where this page was viewed before conversion.",
```

The placeholder will be filled with the name of the goal. Not sure if the change makes sense in that case.

```suggestion
        "ColumnConversionRatePageViewedBeforeDocumentation": "The percentage of all conversions for goal %s where this page was viewed before conversion.",
```

---

The only way to do that in English is to make it
`for the goal "%s"`, which is a bit unnatural wrt. verb, subject, object.
https://ejournals.lib.auth.gr/thal/article/view/5443 https://www.ilovelanguages.com/do-other-languages-have-inverted-verb-order-like-german/
Or "the goal of getting %s done", or something along those lines.